### PR TITLE
[IMP][8.0] Standardize main ReadMe

### LINKED ISSUE #122: README.md not standard
README.md is not structured according to OCA conventions (eg it is missing the addons table).

https://github.com/OCA/maintainer-quality-tools/tree/master/sample_files

### PR DESCRIPTION
**Pull request by lasley**
* Update main ReadMe to OCA guidelines; fixes #122 

One comment inline. I will forward port once this is merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,188 +1,40 @@
...
-
-10. Add your changes to have them versioned.
-
-    ```bash
-    $ git add .
-    ```
-
-11. Commit your changes.
+[//]: # (addons)
+This part will be replaced when running the oca-gen-addons-table script from OCA/maintainer-tools.
```
Comment (lasley) [inline README.md:30]: I am unclear on this part. Does our automation take care of this the first time around, or should I run the script?

Comment (pedrobaeza) [reply README.md:30]: It's done by a script run in OCA server.

**Review by pedrobaeza — APPROVED** [review]

**Comment by lasley** [comment]
@t3ddftw please review

**Review by tedsalmon — APPROVED** [review]
👍  - My only comment is that I prefer ', etc' to a hanging ellipsis but I think that's just personal preference.

**Comment by lasley** [comment]
Thanks for the review @t3ddftw - I agree with `etc` vs the ellipses. TBH I didn't really pay much attention to the text of the Readme, or I would have fixed that. The spacing on them was inconsistent too. Done.

We'll be giving this a major revamping in v9 and 10 after our incubators get merged in. Things have changed quite drastically, and a lot more features now exist.

**Pull request merged**